e2e/selectors.ts

```diff
@@ -111,15 +111,9 @@ export const createSelectors = ({ screen, page, within }: Ctx) => {
       colorLegendItems: async () =>
         (await selectors.chart.colorLegend()).locator("div"),
       legendTicks: async () => {},
-      loaded: async (options: { timeout?: number } = {}) => {
-        await page
-          .locator(`[data-chart-loaded="true"]`)
-          .waitFor({ timeout: 40 * 1000, ...options });
-        const hasMap = (await page.locator("[data-map-loaded]").count()) > 0;
-        if (hasMap) {
-          await page.locator('[data-map-loaded="true"]');
-          await sleep(500); // Waits for tile to fade in
-        }
+      loaded: async () => {
+        await page.waitForLoadState("networkidle");
+        await sleep(1000);
```

Comment:
ls there a particular reason we are sleeping here for 1s?

Reply:
Yes, there are two things that come to my mind, one is that some charts have enter animation (e.g. bar charts) and another is map that take a bit of time (previously we had 500ms) for the tiles to fade in.

Generally it shouldn't really be needed, but some tests interact with the charts themselves, like tooltip tests, so we need to wait for the elements to be visible 👍 I'll add a comment directly in the code so it's clear :)